rebar.config

```diff
@@ -4,7 +4,7 @@
         {snappy, {git, "https://github.com/fdmanana/snappy-erlang-nif.git", {ref, "4e3b861"}}},
         {lz4, {git, "https://github.com/szktty/erlang-lz4.git", {tag, "0.2.4"}}},
         {semver, {git, "https://github.com/nebularis/semver.git", {ref, "c7d509"}}},
-        {uuid, "1.7.2", {pkg, uuid_erl}},
+        {uuid, {git, "git@github.com:SiftLogic/uuid.git", {branch, "master"}}},
```

Comment:
`master` can change better it is to link to tag, in this case `{tag, "sl1.5.2"}` (which is equal to `master`)